modules/mtbseq/tbbwa/tbbwa.nf

```diff
@@ -27,21 +25,21 @@ process TB_BWA {
     gatk-register ${gatk_jar}
 
     mkdir ${genomeFileName}
-   
-    MTBseq --step TBfull --thread ${task.cpus}
-    
-    mv  Amend ./${genomeFileName}/
+    MTBseq --step TBbwa --thread ${task.cpus}
```

Comment:
Might need to update this to `threads` as per their manual 

>  --threads This OPTION is used in TBbwa, TBmerge, TBrefine, TBpile and TBlist. By default, the OPTION is set to 1. The OPTION sets the maximum number of CPUs to use within the pipeline. You can use more than one core in order to execute the pipeline faster. 8 is the current maximum.

Reply:
yes, just fixed that

Reply:
@Mxrcon , good work.

Also, please don't forget to reference the commit here which solves it as a good practice and please allow me to resolve these conversations when I have initiated these :)